test/models/organization_test.rb

```diff
@@ -58,10 +58,10 @@ def organization
   end
 
   test 'requires a URL that exists' do
-    skip """
+    skip '''
```

Comment:
Combine '' and "\n      Read https://www.igvita.com/2006/09/07/validating-url-in-ruby-on-rails/\n      and http://stackoverflow.com/questions/5908017/check-if-url-exists-in-ruby\n    " into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.<br>Combine "\n      Read https://www.igvita.com/2006/09/07/validating-url-in-ruby-on-rails/\n      and http://stackoverflow.com/questions/5908017/check-if-url-exists-in-ruby\n    " and '' into a single string literal, rather than using implicit string concatenation. Or, if they were intended to be separate method arguments, separate them with a comma.